pages/projects/caching.md

```diff
@@ -27,6 +27,5 @@ researchers.
 These [in-production caches](https://indico.cern.ch/event/760850/contributions/3156949/attachments/1724585/2785286/SoCalCache.pdf) have shown to save up to a factor of three of WAN bandwidth compared with
 traditional data management techniques.
 
-##Reports
```

Comment:
`## Reports` should be fine.

Reply:
I noticed that it wasn't displayed as a title but as "##Reports", that's why I removed it

Reply:
If you do want it, you should be able to write 
```markdown
## Reports
```
instead of
```markdown
##Reports
```
and it should work. Some markdown parsers require the space.

Reply:
Ok thanks, I've added it back

Reply:
Thanks!